switch.yml

```diff
@@ -16,6 +16,11 @@
       debug: msg="==== Starting switch playbook ===="
       tags: deploy
 
+    - name: Check if apps_filter is defined
+      fail:
+        msg: "Switch playbook must be used with apps_filter defined"
+      when: apps_filter is not defined
```

Comment:
we could also check than `apps` length is higher than 1 ? If there is only 1 app defined we can skip this task.

WDYT ?

Reply:
Good idea! I also have to fix the deploy playbook to add this.